app/uk/gov/hmrc/fileupload/controllers/envelope.scala

```diff
@@ -47,21 +48,68 @@ object EnvelopeReport {
       expiryDate = envelope.expiryDate,
       status = Some(envelope.status.name),
       metadata = envelope.metadata,
+      constraints = envelope.constraints,
       destination = envelope.destination,
       application = envelope.application,
       files = fileReports
     )
   }
-
 }
 
 case class CreateEnvelopeRequest(callbackUrl: Option[String] = None,
                                  expiryDate: Option[DateTime] = None,
-                                 metadata: Option[JsObject] = None)
+                                 metadata: Option[JsObject] = None,
+                                 constraints: Option[EnvelopeConstraints] = Some(EnvelopeConstraints(100, 100000000L, 1000000L)))
```

Comment:
They both use the static one from `uk.gov.hmrc.fileupload.read.envelope.Envelope.defaultConstraints` now